app/Http/Controllers/ContactController.php

```diff
@@ -2,11 +2,35 @@
 
 namespace App\Http\Controllers;
 
+use App\Http\Request\ContactRequest;
 use Illuminate\Http\Request;
+use Illuminate\Mail\Message;
 
 class ContactController extends Controller
 {
     public function index() {
         return view('contact');
     }
+
+    public function send(ContactRequest $request) {
+        $data = $request->validated();
+        \Log::debug('test', $data);
+
+        \Mail::send(
+            'emails.contact',
+            [
+                'firstName' => $data['first-name'],
```

Comment:
please keep email from User, it is hard to keep communication than you can't respond.